library/ajax/drug_autocomplete/index.php

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
```

Comment:
1. need author/license header
2. also, doesn't this need an include from globals.php ??